getting-started.md

```diff
@@ -141,53 +141,47 @@ Once you navigate back to the `Home` screen, the catalog changes should be refle
 Now that you have a skeleton workload working, you are ready to iterate on your application
 and test code changes on the cluster.
 Tanzu Developer Tools for VSCode, VMware Tanzu’s official IDE extension for VSCode,
-helps you develop & receive fast feedback on the Tanzu Application Platform.
+helps you develop & receive fast feedback on your workloads running on the Tanzu Application Platform.
 
 The VSCode extension enables live updates of your application while it runs on the cluster
 and lets you debug your application directly on the cluster.
 
 For information about installing the pre-requisites and the Tanzu Developer Tools extension, see
-[How to Install the VSCode Tanzu Extension](vscode-extension/install.md).
+[Install Tanzu Dev Tools for VSCode](vscode-extension/install.md).
 
 >**Note:** For this sample app, you must use Tilt v0.23.2 or later
 
-Open the ‘Tanzu Java Web App’ as a project within your VSCode IDE.
+Open the "Tanzu Java Web App" as a project within your VSCode IDE.
 
 In order to ensure your extension helps you iterate on the correct project, you will need to configure its settings:
 
-1. Within VSCode, go to Preferences > Settings > Extensions > Tanzu.
+1. Within VSCode, go to `Preferences` > `Settings` > `Extensions` > `Tanzu`.
 
-1. In the **Local Path** field, enter the path to the directory containing the Tanzu Java Web App.
+1. In the **Local Path** field, enter the path to the directory containing the Tanzu Java Web App. Defaults to current directory.
 
 1. In the **Source Image** field, enter the destination image repository where
 you’d like to publish an image containing your workload source code.
 For example `harbor.vmware.com/myteam/tanzu-java-web-app-source`.
 
 You are now ready to iterate on your application.
 
-
 ### Live update your application
 
 Deploy the application and see it live update on the cluster. Doing so allows you to understand how your code changes will behave on a production-like cluster much earlier in the development process.
 
 Follow these steps:
 
-1. From the Command Palette (⇧⌘P), type in and select **Tanzu: Live Update Start**.
-Tanzu Logs opens up in the Output tab and you will see output from the
-Tanzu Application Platform and from Tilt indicating that the container is being
-built and deployed.
-Because this is your first time starting live update for this application, it
-might take 1-3 minutes for the workload to be deployed and the Knative service
-to become available.
+1. From the Command Palette (⇧⌘P), type in and select `Tanzu: Live Update Start`.
+You will see output from the Tanzu Application Platform and from Tilt indicating that the container is being built and deployed. 
+    - You will also see "Live Update starting..." in the status bar at the bottom right
+    - Because this is your first time starting live update for this workload, it might take 1-3 minutes for the workload to be deployed and the Knative service to become available.
```

Comment:
Remove this? It takes 1-3 minutes any time you start live update if there are any source code changes, not just the first time.

Reply:
@lisamburns Should we just change the text rather than removing it? "Live update can take a few minutes while the workload is deployed and the Knative service becomes available." or something

Reply:
Made the changes on this new PR: https://github.com/pivotal/docs-tap/pull/704/files